pkg/drivers/none/none.go

```diff
@@ -125,6 +131,9 @@ func (d *Driver) GetURL() (string, error) {
 
 // GetState returns the state that the host is in (running, stopped, etc)
 func (d *Driver) GetState() (state.State, error) {
+	runner := command.NewExecRunner(true)
+	d.exec = runner
+
```

Comment:
we initialize a runner here, since it is expected after provisioning phase, for enabling addons